examples/mistral-completion-example/mistral_completion_example.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/llms/mistral"
+)
+
+func main() {
+	llm, err := mistral.New(mistral.WithAPIKey("API_KEY_GOES_HERE"))
+	// note: You would include mistral.WithModel("open-mistral-7b") to use the open-mistral-7b model. Can be overriden when calling the LLM as shown below.
+	// Currently-listed models on Mistral.ai:
+	// open-mistral-7b (aka mistral-tiny-2312)
+	// open-mixtral-8x7b (aka mistral-small-2312) -> DOES NOT seem usable via the mistral-go client library
+	// mistral-small-latest (aka mistral-small-2402)
+	// mistral-medium-latest (aka mistral-medium-2312)
+	// mistral-large-latest (aka mistral-large-2402)
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx := context.Background()
+	completionWithStreaming, err := llms.GenerateFromSinglePrompt(ctx, llm, "Who was the first man to walk on the moon?",
+		llms.WithTemperature(0.8),
+		llms.WithStreamingFunc(func(ctx context.Context, chunk []byte) error {
+			fmt.Print(string(chunk))
+			return nil
+		}),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+	_ = completionWithStreaming
```

Comment:
folks that know go well may understand why this line is here but a comment would be useful